src/test/java/plugins/LogParserTest.java

```diff
@@ -63,6 +62,58 @@ public void trendVisible(){
     }
 
     /**
+     * Test case:
+     * Check for the build to be marked as unstable.
+     */
+    @Test
+    public void checkMarkedUnstableOnWarning() {
+        // Create a special rule set for this test case
+        Resource rule = resource("/logparser_plugin/rules/log-parser-rule-markings");
+        addLogParserRule("error_warning_rule", rule.url.getPath());
+
+        // Create a new freestyle job
```

Comment:
// comments tend to get outdated if the source will be refactored, 
so remove comments that just explain obvious code.